lib/view_model.rb

```diff
@@ -173,6 +173,14 @@ def serialize_to_hash(viewmodel, serialize_context: new_serialize_context)
       Jbuilder.new { |json| serialize(viewmodel, json, serialize_context: serialize_context) }.attributes!
     end
 
+    def serialize_from_cache(views, migration_versions: {}, locked: false, serialize_context:)
+      plural = views.is_a?(Array)
+      views = Array.wrap(views)
+      json_views, json_refs = ViewModel::ActiveRecord::Cache.render_viewmodels_from_cache(views, locked: locked, migration_versions: migration_versions, serialize_context: serialize_context)
```

Comment:
Super long line (even though it's just refactored).

Reply:
Worth fixing